Specs/DateTime/Japanese/DateExtractor.json

```diff
@@ -1,1473 +1,1473 @@
-[
```

Comment:
I assume most changes in this file and the corresponding parser one are just removing the notsupported in dotnet tags and fixing a few entity boundaries?
If so, can you push a new version without the line break changes?